src/gcp_scanner/scanner.py

```diff
@@ -24,14 +24,15 @@
 import sys
 from typing import List, Tuple, Dict, Optional
 
-from . import crawl
-from . import credsdb
+import crawl
```

Comment:
This is required in order to use the tool as a pip module.

Reply:
These have been corrected with upcoming commit.